src/controllers/queryRunner.ts

```diff
@@ -195,19 +196,31 @@ export default class QueryRunner {
 
         let onSuccess = (result) => {
             // The query has started, so lets fire up the result pane
+            QueryRunner._activeQueries.push(this.getFilePath());
+            vscode.commands.executeCommand('setContext', 'mssql.activeQueries', QueryRunner._activeQueries);
+            console.log(this.getFilePath() + ' has been added to activeQueries');
             this.eventEmitter.emit('start', this.uri);
             this._notificationHandler.registerRunner(this, this._ownerUri);
         };
         let onError = (error) => {
             this._statusView.executedQuery(this.uri);
             this._isExecuting = false;
+            QueryRunner._activeQueries = QueryRunner._activeQueries.filter(fileName => fileName !== this.getFilePath());
+            vscode.commands.executeCommand('setContext', 'mssql.activeQueries', QueryRunner._activeQueries);
+            console.log(this.getFilePath() + ' has been removed from activeQueries');
             // TODO: localize
             this._vscodeWrapper.showErrorMessage('Execution failed: ' + error.message);
         };
 
         await queryCallback(onSuccess, onError);
     }
 
+    // helper function for parsing the ownerUri
```

Comment:
Doc comments should be in JS doc form (/** ... */)

But also I don't think a one-liner needs a whole function for it. You can just inline `vscode.Uri.parse(this._ownerUri).fsPath` where you're calling this instead.